awx/settings/local_settings.py.docker_compose

```diff
@@ -221,9 +215,9 @@ LOGGING['handlers']['management_playbooks'] = {'class': 'logging.NullHandler'}
 #}
 
 # Enable the following lines to turn on lots of permissions-related logging.
-#LOGGING['loggers']['awx.main.access']['propagate'] = True
-#LOGGING['loggers']['awx.main.signals']['propagate'] = True
-#LOGGING['loggers']['awx.main.permissions']['propagate'] = True
+#LOGGING['loggers']['awx.main.access']['level'] = 'DEBUG'
+#LOGGING['loggers']['awx.main.signals']['level'] = 'DEBUG'
+#LOGGING['loggers']['awx.main.permissions']['level'] = 'DEBUG'
```

Comment:
Some time ago, I changed these so that they do propagate, so that errors would be logged, but turned down the volume to DEBUG. This change is needed so that the suggested settings changes do what the comment claims.